src/queue/Main.java

```diff
@@ -0,0 +1,40 @@
+package queue;
+
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Queue<Integer> queue = new LinkedList<>();
+
+        // To insert an element in the queue use "offer"
+        for (int i = 0; i < 10; i++) {
+            queue.offer(i);
```

Comment:
Just one note: both `offer` and `add` insert an element into a queue. However, they behave differently.
For an explanation on how the differ, please see: https://softwareengineering.stackexchange.com/a/190270/202859